pkgs/by-name/fa/fastfetch/package.nix

```diff
@@ -162,8 +171,12 @@ stdenv'.mkDerivation (finalAttrs: {
   meta = {
     description = "Like neofetch, but much faster because written in C";
     homepage = "https://github.com/fastfetch-cli/fastfetch";
+    changelog = "https://github.com/fastfetch-cli/fastfetch/releases/tag/${finalAttrs.version}";
     license = lib.licenses.mit;
-    maintainers = with lib.maintainers; [ khaneliman ];
+    maintainers = with lib.maintainers; [
+      luftmensch-luftmensch
+      khaneliman
+    ];
```

Comment:
This change shouldn't be in the nixfmt commit

Reply:
was already addressed

Reply:
sorry, thought it got removed when the new commit was added